packages/connectors/connector-aliyun-dm/src/index.ts

```diff
@@ -91,9 +94,10 @@ const errorHandler = (errorResponseBody: string) => {
     throw new ConnectorError(ConnectorErrorCodes.InvalidResponse, result.error);
   }
 
-  const { Message: errorDescription, ...rest } = result.data;
-
-  throw new ConnectorError(ConnectorErrorCodes.General, { errorDescription, ...rest });
+  throw new ConnectorError(
```

Comment:
Also just double confirm get rid of the reset response data is expected. Only return code and message in the error payload.

Reply:
Tried to balance between readability and security. RequestId and other information can not help end users fix the problem and `RequestId` can be used to get detailed request information on vendors' console page.

Reply:
I want to double check this too. Previously, we don't expose any error data returned from vendor, instead, we save them in the logs table.